resources/dark/iot/thing.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These svgs were copied from https://aws.amazon.com/architecture/icons/, and I manually changed the colors to `#C5C5C5` for dark and `#424242` for light to match colors for the other icons. I was unsure if there was anything additional that needs to be done when adding new icons.

Reply:
That's usually what I do. The toolkit does need a better way to get icons though. The current way doesn't scale at all.